exporter/stackdriverexporter/config.go

```diff
@@ -27,7 +29,9 @@ type Config struct {
 	NumOfWorkers                  int                      `mapstructure:"number_of_workers"`
 	SkipCreateMetricDescriptor    bool                     `mapstructure:"skip_create_metric_descriptor"`
 	// Only has effect if Endpoint is not ""
-	UseInsecure      bool              `mapstructure:"use_insecure"`
+	UseInsecure bool `mapstructure:"use_insecure"`
+	// Timeout for all API calls. If not set, defaults to 12 seconds.
+	Timeout          time.Duration     `mapstructure:"timeout"`
```

Comment:
use definition from https://github.com/open-telemetry/opentelemetry-collector/blob/master/exporter/exporterhelper/common.go#L35 and call WithTimeout with 0 so the logic of adding the timeout to the context is disable because the client sets the timeout in a different way.

Reply:
Done.